### PR TITLE
Cosmetic refinement in Korean translation

### PR DESCRIPTION
**Pull request by jooyunghan**
This fixes many minor issues:

* typo
* spacing
* casing
* newlines
* etc

### CODE REVIEW

File: po/ko.po [hunk 1]
```diff
@@ -1250,7 +1250,7 @@ msgstr "이렇게 하면 [rust-analyzer][1]를 이용해서 특정 심볼이 정
 
 #: src/cargo.md:29
 msgid "Some folks also like to use the [JetBrains][4] family of IDEs, which do their own analysis but have their own tradeoffs. If you prefer them, you can install the [Rust Plugin][5]. Please take note that as of January 2023 debugging only works on the CLion version of the JetBrains IDEA suite."
-msgstr "어떤 사람들은 [Jetbrains][4] 제품군을 선호하기도 합니다. 이 제품들은 rust-analyzer 를 활용하지 않고 IDE 자체적으로 구분분석을 합니다. 만약 이 IDE를 설치하셨다면 [Rust Plugin][5]를 설치하시기 바랍니다. 다만 2023년 1월 기준, 디버깅은 Jetbrains IDEA suite의 CLion 버전에서만 작동한다는 점에 유의하시기 바랍니다."
+msgstr "어떤 사람들은 [JetBrains][4] 제품군을 선호하기도 합니다. 이 제품들은 rust-analyzer 를 활용하지 않고 IDE 자체적으로 구분분석을 합니다. 만약 이 IDE를 설치하셨다면 [Rust Plugin][5]를 설치하시기 바랍니다. 다만 2023년 1월 기준, 디버깅은 JetBrains IDEA suite의 CLion 버전에서만 작동한다는 점에 유의하시기 바랍니다."
```
Comment (jiyongp) [inline po/ko.po:1253]: s/구분분석/구문분석/ ?

File: po/ko.po [hunk 2]
```diff
@@ -1438,7 +1438,7 @@ msgstr ""
 msgid ""
 "You can use <kbd>Ctrl + Enter</kbd> to execute the code when focus is in the\n"
 "text box."
-msgstr "코드 블록에 포커스를 두고 <kbd>Ctrl-Enter</kbd>를 눌러 실행해 볼 수 있습니다."
+msgstr "코드 블록에 포커스를 두고 <kbd>Ctrl + Enter</kbd>를 눌러 실행해 볼 수 있습니다."
```
Comment (jiyongp) [inline po/ko.po:1441]: We need to be consistent on the translation of "block". We currently have "블록" and "블럭". I personally prefer the latter.

**Review by jiyongp — APPROVED** [review]
Thanks a lot. My comments are nits. You may ignore or address later.

File: po/ko.po (hunk 2 above)
Comment (jooyunghan) [reply po/ko.po:1441]: +1

Comment (jooyunghan) [reply po/ko.po:1441]: Added it as a todo in #432

**Pull request merged**